### PR TITLE
feat: Nano App List

### PR DESCRIPTION
**Pull request by ryanio**
#### What does it do?
1. Renames many `client` references to `plugin`
1. Adds a quick launch app list to Nano

Aug-13-2019 18-32-18

### CODE REVIEW

File: preload-webview.js [hunk 1]
```diff
@@ -7,43 +7,47 @@ const PluginHost = remote.getGlobal('PluginHost')
...
+const pluginInterface = plugin => {
   return {
-    name: client.name,
-    displayName: client.displayName,
-    type: client.type,
+    name: plugin.name,
+    displayName: plugin.displayName,
+    type: plugin.type,
+    api: plugin.api,
+    stdinWrite: payload => {
```
Comment (marcgarreau) [inline preload-webview.js]: Assuming this is motivated by Clef? Should it be included in this PR?

Comment (ryanio) [reply preload-webview.js]: thanks for catching

File: ui/nano.html [hunk 2]
```diff
@@ -552,20 +634,38 @@
                 displayName: plugin.displayName,
                 profile: '',
                 version: '',
-                plugin,
+                pluginRef: plugin,
```
Comment (marcgarreau) [inline ui/nano.html]: This breaks a pattern within grid-ui. It is admittedly ugly, but grid-ui is littered with plugin parsing, like `plugin.plugin.config`, for example. `plugin.pluginRef` is better, but I dont love it. Main point: whatever we use should be updated in grid-ui, too. Or abstracted better.

Comment (ryanio) [reply ui/nano.html]: yeah, I've noticed this. the main problem seems to be that we have different interfaces for the plugins, from grid's `PluginHost` to grid's provider to the UI (`preload.js` and `preload-webview.js`) to the store in the UI.

in `grid-ui` we may be able to solve this by renaming some parts to pluginStore and pluginRef can always mean the original interface provided from grid's `PluginHost`.

Comment (marcgarreau) [reply ui/nano.html]: :shipit:

**Comment by evertonfraga** [comment]
That global scrolling on nano can improve a lot. 

I got other weird things about it, a white vertical bar, aligned to the right of nano. 

I’d say the overflow-y should be removed from Body and reused on a downstream div.

**Comment by ryanio** [comment]
oops, yeah I didn't anticipate that - did you see a white vertical bar on win or linux?

I can try putting the overflow-y in a downstream container div as you suggest, but let me know how to recreate the vertical bar first. Seems to be fine for me in my default configuration on mac.

**Pull request merged**